src/components/extensionTopContent/top-contents/GlobalNavigationBar.jsx

```diff
@@ -4,25 +4,39 @@ import { useContext } from "react";
 
 import ExtensionContext from "../../../context/ExtensionContext";
 
-function GnbButton({ iconShapeType }) {
+function GnbButton({ iconShapeType, onClick }) {
   return (
     <button className="w-8 text-lg">
-      <FontAwesomeIcon icon={iconShapeType} />
+      <FontAwesomeIcon
+        icon={iconShapeType}
+        onClick={onClick}
+      />
     </button>
   );
 }
 
 export default function GlobalNavigationBar() {
   const { bookmarkList } = useContext(ExtensionContext);
 
+  const hendleOnClickOpenOptionPage = () => {
+    if (chrome.runtime.openOptionsPage) {
+      chrome.runtime.openOptionsPage();
+    } else {
+      window.open(chrome.runtime.getURL("options.html"));
+    }
+  };
+
   return (
     <div className="w-full text-white flex mb-3 items-center">
       <p className="grow">
         <span className="text-2xl">{bookmarkList.length}</span>
         <span className="text-sm">개의 북마크</span>
       </p>
       <div className="flex text-white">
-        <GnbButton iconShapeType={faDesktop} />
+        <GnbButton
+          iconShapeType={faDesktop}
+          onClick={hendleOnClickOpenOptionPage}
```

Comment:
```suggestion
          onClick={handleOnClickOpenOptionPage}
```
함수명에 오타가 포함된 것 같습니다. 수정하면 좋을 것 같습니다. 감사합니다!

Reply:
위에 오타 확인 해주신 부분과 같은 함수로 확인하였습니다. 윗 함수 수정하면서 함께 수정하도록 하겠습니다. 
감사합니다.